bot.py

```diff
@@ -862,6 +862,29 @@ def to_html():
     pass
 
 
+def split_json_file(data: dict, output_path: str, max_size: int = MAX_FILE_SIZE):
+    messages = data["messages"]
+    part_count = 1
+    current_data = {"messages": []}
+
+    for msg in messages:
+        current_data["messages"].append(msg)
+        serialized = json.dumps(current_data, default=str)
```

Comment:
`serialized` is not clear. serialized what? I suggest use `serialized_pages` or sth like that.